snapcraft/plugins/catkin.py

```diff
@@ -1062,14 +1073,14 @@ class _Catkin:
     def __init__(
```

Comment:
minor, but can we add some type annotations to __init__?

Reply:
Sure, done.